ansible/templates/zapd_block_height.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+dest_email=alerts@zap.me
+remote_block=`curl -s https://testnode1.wavesnodes.com/blocks/height | jq '.["height"]'` 
+local_block=`curl -s localhost:6869/blocks/height | jq '.["height"]'`
+max_height_diff=2
+
+### Remote block would be same/higher than local block.
+num=$(( $remote_block - $local_block ))
```

Comment:
its actually possible for us to get ahead of the specific remote node we are talking to because its a distributed network.. so make this the absolute value of the difference